src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(101);
+        Ball randomBall = new Ball(getRandomColor(), randomNumber);
```

Comment:
You can directly return the new 'Ball' object without assigning it to the 'randomBall' variable, as the temporary variable is unnecessary here.